dags/hivemind_website_ingestion.py

```
@@ -0,0 +1,98 @@
+    catchup=False,
+    max_active_runs=1,
+) as dag:
+
+    @task
+    def get_mongo_discord_communities() -> list[dict[str, str | datetime | list]]:
+        """
+        Getting all communities having discord from database
+        this function is the same with `get_discord_communities`
+        we just changed the name for the pylint
+        """
+        communities = ModulesDiscord().get_learning_platforms()
```

Ruff

Undefined name `ModulesDiscord`

(F821)